homeassistant/components/sensor/fronius.py

```diff
@@ -0,0 +1,158 @@
+"""
+Support for Fronius devices.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/sensor.fronius/
+"""
+import asyncio
+from concurrent.futures._base import TimeoutError
+from datetime import timedelta
+import logging
+
+from aiohttp.client_exceptions import ServerDisconnectedError
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_HOST, CONF_SCAN_INTERVAL
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_time_interval
+import voluptuous as vol
```

Comment:
please put 3rd party imports above home assistant imports

Reply:
Hm... this order has been generated from isort.
Somewhere else @fabaff asked me to use isort for the imports.
What's the right way?

Reply:
This is not sorted by `isort`. It would look like this

```python
...
import logging

from aiohttp.client_exceptions import ServerDisconnectedError
import voluptuous as vol

from homeassistant.components.sensor import PLATFORM_SCHEMA
from homeassistant.const import CONF_HOST, CONF_SCAN_INTERVAL
...
```